.golangci.yml

```diff
@@ -103,7 +101,6 @@ linters:
     - unconvert
     # - unparam
     - unused
-    - varcheck
```

Comment:
For the 3 removed linters, golangci warned: "The owner seems to have abandoned the linter.  Replaced by unused."